docker-compose.override.yml

```diff
@@ -39,7 +39,7 @@ services:
     user: node
     working_dir: /usr/src/app
     ports:
-      - 3000:3000
+      - 4201:3000
```

Comment:
Changed the viewer port to `4201` in both development and production to make it easier to remember.